dbt/seeds/ccao/ccao.commercial_subclass_1.csv

```diff
@@ -0,0 +1,7 @@
+code,description
```

Comment:
Does it make sense to you to add these descriptions as seeds, similar to `reascd`? If so, what do you think about the name of the seeds?